src/tracing/CMakeLists.txt

```diff
@@ -9,7 +9,9 @@ file(MAKE_DIRECTORY ${header_dir})
 add_custom_target(tracepoint_libraries)
 
 file(GLOB tps "*.tp")
+message(STATUS "all tps: ${tps}")
 foreach(tp ${tps})
+  message(STATUS "process ${tp}")
```

Comment:
i think we can drop these debug messages

Reply:
ok